src/api/spec/features/beta/webui/maintenance_workflow_spec.rb

```diff
@@ -20,10 +20,17 @@
   let!(:another_branched_package) { create(:package_with_file, project: branched_project, name: 'cacti.openSUSE_11.5_Update', file_name: 'DUMMY_FILE', file_content: 'boo#12345') }
   let(:update_project) { create(:update_project, maintained_project: project, name: "#{project}:Update") }
   let(:another_update_project) { create(:update_project, maintained_project: another_project, name: "#{another_project}:Update") }
+  let!(:patchinfo) { user.run_as { create(:patchinfo, project_name: branched_project.name, package_name: '_patchinfo') } }
   let(:bs_request_without_patchinfo) do
     create(:bs_request_with_maintenance_incident_actions, source_project_name: branched_project.name, source_package_names: [package.name], target_project_name: maintenance_project.name,
                                                           target_releaseproject_names: [update_project.name, another_update_project.name])
   end
+  let(:bs_request_with_patchinfo) do
```

Comment:
@hellcp-work would move this to the context `maintenance request with patchinfo`

Reply:
same of the `patchinfo`